libs/dex/src/main/java/io/crate/data/Bucket.java

```diff
@@ -49,4 +51,8 @@ public String toString() {
      * @return the number of rows
      */
     int size();
+
+    default Stream<Row> stream() {
```

Comment:
Since to my understanding this is used only for testing, shouldn't we create a special `BucketAssertion` with a method to assert the rows?

Reply:
Inlined it for now.